xarray/core/ops.py

```diff
@@ -187,8 +187,6 @@ def _call_possibly_missing_method(arg, name, args, kwargs):
         method = getattr(arg, name)
     except AttributeError:
         duck_array_ops.fail_on_dask_array_input(arg, func_name=name)
-        if hasattr(arg, 'data'):
-            duck_array_ops.fail_on_dask_array_input(arg.data, func_name=name)
```

Comment:
What is this block doing? Tests are passing without this.

Reply:
I'm sure we would get a bug report if it matters!

But for now it's probably best to leave it just in case, and save removing it for a refactor where we eliminate the `_call_possibly_missing_method` function entirely.